recipes/scikit-build-core/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "scikit-build-core" %}
+{% set version = "0.1.0b2" %}
```

Comment:
This should probably target pre-release builds: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

Reply:
There is no non-prerelease build, so I was thinking this is best. And the main reason it's still pre-release is so I can verify it works on conda-forge first.

Reply:
Ahh ok. In that case, you should still be able to make a prerelease build and test it by adding the respective flags but let me ask core, though, to make sure. @ocefpaf, what do you think?

Reply:
It's 0.1.0 now!